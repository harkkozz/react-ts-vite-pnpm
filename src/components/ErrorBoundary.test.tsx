import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';

import ErrorBoundary from './ErrorBoundary';

describe('Error boundary', () => {
  it('Should match snapshot', () => {
    const { container } = render(<ErrorBoundary />);

    expect(container).toMatchSnapshot();
  });
});
