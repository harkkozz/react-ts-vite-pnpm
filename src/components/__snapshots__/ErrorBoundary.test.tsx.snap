// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Error boundary > Should match snapshot 1`] = `
<div>
  <div>
    ErrorBoundary
  </div>
</div>
`;
